### ISSUE TITLE
Right click open with Notepad++ does nothing

### ISSUE THREAD

**Issue by dudeedud4** [notepad-plus-plus/notepad-plus-plus#13845]
### Description of the Issue

When right clicking a file and choosing "edit with notepad++" nothing happens.

### Steps to Reproduce the Issue

1. Right-click .txt/file
2. Click "Open with Notepad++"
3. See that nothing happens

### Expected Behavior

I expected the editor to launch.

### Actual Behavior

Nothing at all.

### Debug Information

Notepad++ v8.5.4   (64-bit)
Build time : Jun 17 2023 - 20:42:45
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.3086
Current ANSI codepage : 1252
Plugins : 
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)

**Comment by donho** [comment]
@dudeedud4 
"Open with Notepad++" is not the entry name for Notepad++ official release.
Could you do a screenshot of your explorer context menu?

**Comment by dudeedud4** [comment]
@donho I can't believe I'm going to be that guy but... It's working now. Unfortunately I won't be able to take a picture as when I went to do just that the option appears to be working. And you are correct, it says "Edit with Notepad++", sorry bout that mistyped and didn't realize it. I'll just close this and reopen if need be.

It's possible a restart fixed it but I'm not too sure, haven't tried the context menu option until I saw this reply to the issue.
